components/ide-service-api/ide.proto

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+syntax = "proto3";
+
+package ide_service_api;
+
+option go_package = "github.com/gitpod-io/gitpod/ide-service/api";
+option java_package = "io.gitpod.ideservice.api";
+
+service IDEService {
+    rpc GetConfig(GetConfigRequest) returns (GetConfigResponse) {}
+}
+
+message GetConfigRequest {}
+
+message GetConfigResponse {
+    string content = 1;
```

Comment:
I think for first iteration is alright 👍  let's bring it to production and switch server to resolve from here, only after that we should move type definitions here and do parsing in our service

Reply:
I hoped that we could reuse types in installer if we define them here or just in go

Reply:
Yes, we can move types to `go` folder under `ide-service-api`  in next PR, for other components to reuse them